linode/manager
# refactor: [M3-6081] - Use region `label` from `/v4/regions` instead of `dcDisplayNames` constant

## Description 📝

- Uses the API to derive region names rather than the constant `dcDisplayNames`

## Todo ☑️

## How to test 🧪

- Run e2e tests
- Verify that region names in Cloud Manager match what production shows
